.github/workflows/gradle.yml

```diff
@@ -0,0 +1,25 @@
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ cdSetupBranch ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@937999e9cc2425eddc7fd62d1053baf041147db7
+        with:
+          arguments: build
```

Comment:
A little confused why you need this separate workflow when you have the same steps in `ruby.yml`? Did you intend this to run your Java tests? If so, you need to do more than just `gradle build`.

Reply:
I added a step to run the tests!